optaplanner-quickstart/src/main/java/org/acme/optaplanner/DemoDataGenerator.java

```diff
@@ -0,0 +1,193 @@
+package org.acme.optaplanner;
```

Comment:
Any reason not to have this bean in its own package?

Reply:
Do we have any other examples that have data access specific code? What package name do they use?

Reply:
Proposal for subpackage:
A) persistence
B) bootstrap
C) lifecycle
X) Keep it in the top package (status quo)

Wdyt?

Reply:
I do like B)